src/main/java/kostka/moviecatalog/controller/AdministrationController.java

```diff
@@ -135,6 +135,23 @@ public String createUser(final @Valid UserFormDto dto,
         }
     }
 
+    /**
+     * Method for deleting existing user from db..
+     * @param userId id of the deleted user.
+     */
+    @GetMapping("/users/delete/{userId}")
+    public String deleteUser(final @PathVariable Long userId) {
+        LOGGER.info("delete user with id '{}' request", userId);
+
+        try {
+            userService.deleteUser(userId);
+            rabbitMqSender.sendRefreshAdminRequestToQueue();
+        } catch (Exception e) {
```

Comment:
minor: can rabbitMqSender throw an exception? If yes, logged error could be misleading.

Reply:
rabbitMQSender cannot send exception ;) 
but yeah, I can extract that from the try block :)

Reply:
It would be nice I guess. Or you can throw custom exception from userService. It is up to you.